can_bus/json/FSM/FSM_tx.json

```diff
@@ -1,181 +1,181 @@
 {
-	"Vitals": {
-		"msg_id": 304,
-		"cycle_time": 100,
-		"description": "FSM Heartbeat signal",
-		"signals": {
-			"Heartbeat": {
-				"bits": 1
-			},
-			"State": {
-				"enum": "State_Machine"
-			}
-		}
-	},
-	"AlertsContext": {
-		"msg_id": 301,
-		"cycle_time": 1000,
-		"signals": {
-			"WatchdogTimeoutTaskName": {
```

Comment:
What's with all the whitespace diffs?

Reply:
There was some inconsistent indentation before hand... I can undo the whitespace diff, keep it, or merge it in a new PR... LMK what you think...